x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result.scss

```diff
@@ -0,0 +1,55 @@
+.appSearchResult {
+  overflow: hidden;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+
+  &__contentWrap {
+    width: 100%;
+    padding: $euiSize;
+    display: grid;
+    grid-template-rows: 1fr $euiSizeXS * 5;
+    border-right: $euiBorderThin;
+  }
+
+  &__contentInner {
+    display: grid;
+    grid-gap: $euiSizeS;
+    overflow: hidden;
+  }
+
+  &__fieldsetContainer {
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__hiddenFieldsIndicator {
+    font-size: $euiFontSizeXS;
+    color: $euiColorDarkShade;
+    align-self: end;
+    padding-top: 0;
```

Comment:
See above comment

```suggestion
    margin-top: $euiSizeS;
```

Reply:
Something I totally meant to put in this review comment set but forgot to - a lot of my extra cleanup comments are only possible due to the already-awesome cleanup efforts you've done - thanks so much for them!! Huge fan of the flex vs grid cleanup you've done